.github/workflows/ci.yaml

```
@@ -50,3 +50,53 @@ jobs:
           labels: ${{ steps.docker_meta.outputs.labels }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          outputs: type=docker,dest=/tmp/workload-standard-${{ github.sha }}.tar
+      - name: Upload artifact
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: docker-image
+          path: /tmp/workload-standard-${{ github.sha }}.tar
+          retention-days: 1
+
+  scan:
+    runs-on: [self-hosted, docker, integration]
+    needs: build
+    strategy:
+      matrix:
+        include:
+          - severity: "MEDIUM,HIGH"
+            exit-code: "0"
+          - severity: "CRITICAL"
+            exit-code: "1"
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Download artifact
```

```suggestion
      - name: Download artifact
        if: ${{ github.event_name == 'pull_request' }}
```

Seems like this step will only succeed for PR.

---

good catch